AppVeyor/AppVeyorInstall.ps1

```diff
@@ -29,8 +34,9 @@ Write-Host "Installed NuGet version '$($pkg.version)'"
 # Install Modules                 #
 #---------------------------------#
 [version]$ScriptAnalyzerVersion = '1.8.1'
+[version]$PesterVersion = '4.10.1'
 Install-Module -Name 'PSScriptAnalyzer' -Repository PSGallery -Force -ErrorAction Stop -MaximumVersion $ScriptAnalyzerVersion
-Install-Module -Name 'Pester' -SkipPublisherCheck -Repository PSGallery -Force -ErrorAction Stop
+Install-Module -Name 'Pester' -SkipPublisherCheck -Repository PSGallery -Force -ErrorAction Stop -MaximumVersion $PesterVersion

```

Comment:
Why are we setting this here? What do later versions break?

Reply:
The version 5.x of Pester has a lot of breaking changes. I was unable to get Pester 5 working with the current code. 

For breaking changes see: https://github.com/pester/Pester/releases/tag/5.0.0#breaking-changes